core-plugin/testSrc/com/google/gct/idea/debugger/CloudDebuggerClientTest.java

```diff
@@ -0,0 +1,64 @@
+package com.google.gct.idea.debugger;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.services.clouddebugger.Clouddebugger.Debugger;
+import com.google.gct.login.CredentialedUser;
+import com.google.gct.login.GoogleLogin;
+import com.google.gdt.eclipse.login.common.GoogleLoginState;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.LinkedHashMap;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CloudDebuggerClientTest {
+
+  @Before
+  public void setUp() {
+    GoogleLogin mockLogin = Mockito.mock(GoogleLogin.class);
```

Comment:
Can we use the MockGoogleLogin class here? So we don't have to make "setInstance" public. I'd rather that not be visible as an option for non-test development.